doc/topics/development/conventions/formulas.rst

```diff
@@ -79,12 +79,12 @@ or zip file of the repository. The directory structure is designed to work with
 
         mkdir -p /srv/formulas
         cd /srv/formulas
-        git clone https://github.com/saltstack-formulas/apache-formula.git
+        git clone https://github.com/saltstack-formulas/apache-formula.git /srv/formulas/apache-formula
 
         # or
 
-        mkdir -p /srv/formulas
-        cd /srv/formulas
+        mkdir -p /srv/formulas/apache-formula
+        cd /srv/formulas/apache-formula
```

Comment:
This will still do the same thing, because `apache-formula` is the top level file in the archive.